x-pack/docs/en/security/operator-privileges/configure-operator-privileges.asciidoc

```diff
@@ -0,0 +1,61 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[configure-operator-privileges]]
+=== Configure operator privileges
+
+==== Enable operator privileges
+Operator privileges need to be enabled explicitly on each node in the cluster.
+This is done by configure the `elasticsearch.yml` file:
+
+[source,yaml]
+----------------------------
+xpack.security.operator_privileges.enabled: true
+----------------------------
+
+If the node is already running before the above change is made, you need restart
+it for the feature to take effect.
+
+WARNING: The feature needs to be either enabled or disabled consistently across all nodes
+in a cluster. Otherwise, you can get inconsistent behaviours depending on which node
+a request hits first and where it gets executed.
+
+==== Designate operator users
+Operator users are just normal {es} users with special privileges to perform
+operator-only functionalities. They are specified in an `operator_users.yml` file located
+in `ES_PATH_CONF`. Similar to <<file-realm-configuration,other security config files>>,
+the `operator_users.yml` file is local to a node and not globally to the cluster.
+This means, in most cases, the same file should be distributed/copied to all nodes in
+a cluster.
+
+The `operator_users.yml` file defines a set of criteria that an authenticating
+user must match to be considered as an operator.
+The following snippet shows an example of such a file:
+
+[source,yaml]
+-----------------------------------
+operator: <1>
+  - usernames: ["system_agent_1","system_agent_2"] <2>
+    realm_type: "file" <3>
+    realm_name: "file1" <4>
+    auth_type: "realm" <5>
+-----------------------------------
+<1> A fixed value of `operator` to signal the beginning of the definition.
+<2> A list of usernames allowed for operator users. This field is mandatory.
+<3> The type of the authenticating realm allowed for operator users. The default and only
+    acceptable value is <<file-realm,`file`>>.
+<4> The name of the authenticating realm allowed for operator users. For the `file` realm, the name is ignored.
```

Comment:
If the `realm_type` has to be file, and `realm_name` is ignored for file, doesn't that mean `realm_name` is always ignored, and we can drop it from the docs?

Reply:
I think this question is still outstanding.

Reply:
Somehow I missed this one. `realm_name` is now removed.